Sales/src/main/java/com/kennuware/sales/services/StubCallerServiceExample.java

```diff
@@ -0,0 +1,53 @@
+package com.kennuware.sales.services;
+
+
+import org.apache.http.HttpEntity;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.ClientProtocolException;
+import org.apache.http.client.ResponseHandler;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.util.EntityUtils;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+
+import java.io.IOException;
+
+/**
+ * Created by Ryan on 10/24/2016.
+ */
+public class StubCallerServiceExample {
+
+    public void testStub() {
```

Comment:
This is the example to make the call to the stub project. Pretty straightforward. Must have the stub project running for this to work.